mne/beamformer/_lcmv.py

```diff
@@ -589,6 +576,8 @@ def tf_lcmv(epochs, forward, noise_covs, tmin, tmax, tstep, win_lengths,
     .. [2] Sekihara & Nagarajan. Adaptive spatial filters for electromagnetic
            brain imaging (2008) Springer Science & Business Media
     """
+    # TODO this has an entirely different code path, need to decide if we
+    # can unify it, or deprecate this, etc.
```

Comment:
I am pro deprecation. There are better methods now (that we have better computational power etc) than chopping data and computing covariance matrices on those snippets ... e.g., using Hilbert-data per frequency band and getting good time resolution, or using DICS with many frequencies and getting good frequency resolution. 
If someone still wants to do it, I feel it would be straight-forward to write a loop over time windows and frequency bands and do covariance computation and beamformer in that loop.

Reply:
Okay I'll deprecate then